travvy/src/components/Home.vue

```diff
@@ -37,9 +37,9 @@
     <!-- data binding dates, corresponding data object below -->
     <label for="Dates">     Select Departure Date:</label>
     <!-- user selects dates they are travelling in -->
-      <input type="Date" min="2020-11-09" max="2022-12-31" class="dates input" v-model="startDate">
+      <input type="Date" min="2020-11-23" max="2022-12-31" class="dates input" v-model="startDate">
     <label for="Dates">     Select Return Date:</label>
-      <input type="Date" min="2020-11-09" max="2022-12-31" class="dates input" v-model="endDate">
+      <input type="Date" min="2020-11-23" max="2022-12-31" class="dates input" v-model="endDate">
```

Comment:
It will be interesting to see if we can make this dynamic at some point - so we do not have to set the current date as the minimum date!

Reply:
Definitely, we can tackle this in our upcoming sprint 4!